### ISSUE TITLE
Failed parsing when plib material keyword is used

### ISSUE THREAD

**Issue by dodu94** [idaholab/MontePy#477]
**Describe the bug**

Hi, I'm keeping experimenting with MontePy. This time I had a failed parsing when using a PLIB keyword (if you remove it from the sample input the parsing is successful). I was not sure if these "niche" additional material keys were supported, but than I checked the lexer and it seems that they are:

![image](https://github.com/user-attachments/assets/3a999a56-e19f-4753-ab5d-21e5dc17c16d)

**To Reproduce**

attempt parsing the mcnp input attached

**Error Message (if any)**

``` python
Traceback (most recent call last):
  File "d:\DATA\laghida\Desktop\test_montepy\read.py", line 4, in <module>
    problem = montepy.read_input("test.i")
  File "C:\ProgramData\Anaconda3\envs\montepy\lib\site-packages\montepy\input_parser\input_reader.py", line 29, in read_input
    problem.parse_input(replace=replace)
  File "C:\ProgramData\Anaconda3\envs\montepy\lib\site-packages\montepy\mcnp_problem.py", line 294, in parse_input
    trailing_comment = obj.trailing_comment
  File "C:\ProgramData\Anaconda3\envs\montepy\lib\site-packages\montepy\mcnp_object.py", line 262, in trailing_comment
    return self._tree.get_trailing_comment()
  File "C:\ProgramData\Anaconda3\envs\montepy\lib\site-packages\montepy\input_parser\syntax_node.py", line 197, in get_trailing_comment
    return tail[1].get_trailing_comment()
  File "C:\ProgramData\Anaconda3\envs\montepy\lib\site-packages\montepy\input_parser\syntax_node.py", line 2121, in get_trailing_comment
    return tail[1].get_trailing_comment()
  File "C:\ProgramData\Anaconda3\envs\montepy\lib\site-packages\montepy\input_parser\syntax_node.py", line 197, in get_trailing_comment
    return tail[1].get_trailing_comment()
AttributeError: 'NoneType' object has no attribute 'get_trailing_comment'
```

**MCNP input file snippet**

```
1   1 -1 1 -2
 
1    pz 0
2    pz 2

m1      8016.71c  2.6999999-02
        8017.71c  9.9999998-01
        plib=84p
NPS 1
```

**Version**

 - Version 0.3.3.dev11+g55d0c4d

Thanks for the great package you are building!

**Comment by MicahGale** [comment]
Thanks for the bug report. This seems to be a typing error that something that should never be None is None, and not the fact that `plib` is a keyword or not. 

This issue and others is making me second guess not being nearly as strict with the `montepy.input_parser.syntax_node` package.

**Comment by MicahGale** [comment]
Fixed in 0.3.3.
